tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -813,6 +813,11 @@ async def test_did_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment):
     for _ in range(3):
         await farm_transaction_block(full_node_api, wallet_1_node)
 
+    async def num_wallets() -> int:
+        return len(await wallet_2_node.wallet_state_manager.get_all_wallet_info_entries())
+
+    await time_out_assert(30, num_wallets, 2)
```

Comment:
this fixed an issue caused by supporting concurrent readers. Without this, the transaction won't have been committed by the time we call `get_all_wallet_info_entries()`, and we won't see the new wallet.